src/components/CalendarMonth.jsx

```diff
@@ -203,7 +203,7 @@ class CalendarMonth extends React.PureComponent {
               isVisible,
             })
           ) : (
-            <strong>
+            <strong aria-live="polite">
```

Comment:
Recommend also adding `aria-atomic="true"` so that the whole phrase gets read out

Reply:
@backwardok ooo thanks for the recommendation! I just added that as well :)